changelog/fragments/remove-exec-entrypoint.yaml

```
@@ -0,0 +1,6 @@
+entries:
+  - description: >
+      The `operator-sdk` command no longer supports `exec-entrypoint` subcommand for ansible and helm operators. The entrypoints
+      for ansible/helm base images are from separate binaries present inside `cmd/` folder.
```

The fragments are used to generate the CHANGELOG and migration guide `The text `The entrypoints  for ansible/helm base images are from separate binaries present inside `cmd/` folder.` shows make no sense for end-users. 

Also, should not users change their projects and update their Dockerfile after that?

```suggestion
      **(Valid just for Ansible/Helm-based Operators). The subcommand `operator-sdk exec-entrypoint` was removed. To use SDK to runs as Ansible/Helm-based operator from inside of a Pod in a cluster use go run `cmd/<type>-operator/main.go` instead of it. 
```